bindata/assets/alerts/kube-apiserver-webhook.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: webhooks
+  namespace: openshift-kube-apiserver
+spec:
+  groups:
+    - name: webhooks
+      rules:
+        - alert: HighAdmissionWebhookLatency
+          annotations:
+            summary: Webhook Latency is high.
+            description: >-
+              The average latency due to the {{ $labels.name }} (type="{{ $labels.type }}") webhook is over 1s.
+          expr: |
+            (sum(rate(apiserver_admission_webhook_admission_duration_seconds_sum[5m]))by(name)-sum(rate(apiserver_admission_webhook_admission_duration_seconds_bucket{le="1"}[5m]))by(name))>0
+          for: 10m
+          labels:
+            namespace: openshift-kube-apiserver
+            severity: critical
```

Comment:
Because this alert is critical, we have to introduce a runbook to it.

Reply:
the runbook mentions that this condition is slowing down API server requests - should we confirm this statement within the `expr` and also detect that PUTs/POSTs are indeed increasing in duration?